### ISSUE TITLE
iBOT NaN Loss

### ISSUE THREAD

**Issue by ChuaHanChong** [valeoai/Franca#12]
Hi,  
I really appreciate your great work.  

I am using your implementation on my own data, and I would like to double-check with you regarding the high-value iBOT head, which causes the Sinkhorn-Knopp centering to produce NaN values. Referring to this [code](https://github.com/valeoai/Franca/blob/main/franca/loss/mrl_ibot_patch_loss.py#L60), `sum_Q` becomes `inf` if `Q` has very large values. I confirmed that this is due to the prototype logits from the iBOT head being excessively high.  

When I look at your implementation of the [forward function](https://github.com/valeoai/Franca/blob/f5cc9a5bcd84b384f63cfd7091a1399d79ee6905/franca/layers/mrl_dino_head.py#L97-L112) for the DINO head (which has the same architecture as the iBOT head), it does not include normalization as implemented in the [forward function](https://github.com/facebookresearch/dinov2/blob/b8931f7bf91576930313be2c6d6af376033b35f0/dinov2/layers/dino_head.py#L36-L41) for DINOv2.  

I would greatly appreciate your guidance on this matter.  @shashankvkt

**Comment by ChuaHanChong** [comment]
Yes, NaN is caused due the numerical overflow. 
Thanks for your help.
